### ISSUE TITLE
invalid SRU percentage in  Node Resources

### ISSUE THREAD

**Issue by 0oM4R** [threefoldtech/tfgrid-sdk-ts#188]
### Description

the SRU values in some nodes  are over 100%
![Screenshot from 2023-04-26 11-58-17](https://user-images.githubusercontent.com/62248851/234542202-23a7aa5b-0395-4516-a2e7-8bb8048fbf8d.png)
- node 11
  ![image](https://user-images.githubusercontent.com/62248851/234542325-43b33df7-30b5-43c1-97b0-7fb94f9f3ef0.png)
- node 17
   ![image](https://user-images.githubusercontent.com/62248851/234542959-8697063a-2730-4537-82ff-1fabbf0aa014.png)

### Steps to reproduce

- Navigate to https://dashboard.dev.grid.tf/explorer/nodes
- Click on node 11,14 or 17 from the nodes table.

**Comment by 0oM4R** [comment]
The calculation here is totally fine; this needed to be fixed in gridProxy, or we can use the statistics endpoint after it gets fixed.

**Comment by AhmedHanafy725** [comment]
this sound like resolved for this node as the node is freed up. but if someone deployed some workload on it, the issue will happen again. so I think the usage should be calculated from the statistic endpoint as it gets the exact usage at that moment and the call is already done to get the number of workloads and deployments on the node so no more requests will be done.

**Comment by 0oM4R** [comment]
Update:
almost done,just need to handle fetch errors

**Comment by 0oM4R** [comment]
Status: 
PR #248  ready for review

**Comment by 0oM4R** [comment]
Status:
Merged waiting for release

**Comment by mohamedamer453** [comment]
Verified on devnet.

I checked the same nodes that had the invalid sru percentage and it's now displayed normally even when the nodes are almost full.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/cb11abe8-d8c7-4554-9384-c6aba66aa2e6)

and also the calculations is now returned from the statistics endpoint.

fore example if we calculate the sru usage for node 17 from the statistics endpoint which is supposed to be the current usage we will find that it's the same number displayed in the dashboard.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/537f36c8-984a-4a31-ab32-d549f9982ded)

```
((421930205184 + 16106127360) / 512110190592) * 100 = 85.5355%
```
